### PR TITLE
feat: apply i18n to tutorial chapter buttons

### PR DESCRIPTION
**Pull request by n0hack**
## What is this PR?

i have implemented i18n to the chapter buttons on the tutorial page. additionally, added a new depth in `config.i18n` to clearly separate where it is used. also I am considering whether changes are needed for existing properties like `i18n.continueReading` due to the modification in the hierarchy.

Welcome any feedback!

```json
"i18n": {
  "next": {
    "en": "Next Page",
    "ja": "次のページ",
    "de": "Nächste Seite",
    "fr": "Page Suivante",
    "es": "Siguiente Página",
    "tr": "Sonraki Sayfa",
    "it": "Pagina Successiva",
    "pt-br": "Próxima Página",
    "zh": "下一页",
    "kr": "다음",
    "ru": "Следующая страница"
  },
  "previous": {
    "en": "Previous Page",
    "ja": "前のページ",
    "de": "Vorherige Seite",
    "fr": "Page Précédente",
    "es": "Pagina Anterior",
    "tr": "Önceki Sayfa",
    "it": "Pagina Precedente",
    "pt-br": "Página Anterior",
    "zh": "上一页",
    "kr": "이전",
    "ru": "Предыдущая страница"
  },
  "continueReading": {
    "en": "Continue reading",
    "de": "Weiter lesen",
    "zh": "继续阅读",
    "es": "Seguir leyendo",
    "ru": "Продолжить чтение"
  },
  "tutorial": {
    "previous": {
      "en": "Previous",
      "kr": "이전"
    },
    "help": {
      "en": "Help",
      "kr": "정답 확인"
    },
    "next": {
      "en": "Next",
      "kr": "다음"
    }
  }
}
```

## Screenshot
image

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -52,6 +52,20 @@
 			"zh": "继续阅读",
 			"es": "Seguir leyendo",
 			"ru": "Продолжить чтение"
+		},
+		"tutorial": {
+			"previous": {
```
Comment (rschristian) [inline src/config.json:57]: I think we can just reuse the existing `previous` & `next` -- I get (in English, anyways) it's a slight change but that should be fine and I think still makes sense?

Comment (n0hack) [reply src/config.json:57]: @rschristian Great feedback, i agree! In that case, i'll reuse the existing two properties and only add `help` property under the `tutorial` context.

Is it okay to do it this way? `help` may be used on other pages, placing it within the `tutorial` contexts seems better as their purposes differ.

**Comment by n0hack** [comment]
@rschristian I've include your feedback. Please review it

File: src/config.json (hunk 1 above)
Comment (rschristian) [reply src/config.json]: > Is it okay to do it this way? help may be used on other pages, placing it within the tutorial contexts seems better as their purposes differ.

I think that makes a lot sense, sounds great to me!

**Review by rschristian — APPROVED** [review]
Thank you so much!

**Pull request merged**